### PR TITLE
Feat/how it works section

### LINKED ISSUE #7: Create How it Works Section
## 📄 Issue Description
Implement the How it Works section of the landing page based on the Figma design. This section showcases three stages: Register & Setup, Create and ShareLink, and Receive Payments.

## 🔍 Steps

1. Analyze the Figma design for the "How it Works" section.
2. See how the components of the current landing page are implemented and created.
3. Create a reusable component for each stage within the "How it Works" section.
4. Add appropriate icons or visuals for each stage.
5. Ensure the layout is responsive and visually consistent.

## ✅ Acceptance Criteria

- The "How it Works" section matches the Figma design.
- Each stage (Register & Setup, Create and ShareLink, Receive Payments) is clearly represented with corresponding visuals or icons.
- Section is fully responsive and accessible.
- Components must be separated in their individual folder per section. In this case, use the how-it-works folder in components.

## 🌎 References
[Figma Design](https://github.com/PayStell/paystell-landing/blob/main/landingPage.md)

### PR DESCRIPTION
**Pull request by kimcascante**
# Pull Request Overview

## 📝 Summary
This PR implements an update for the "How it Works" section in third step to include animation more visible

### 🔗 Related Issues
- Closes #7 

## 🔄 Changes Made
I tried to update SVG image to make it fit as the figma design, but I just could change animation to be more visible, and I couldn't avoid the background image to fit in any resolution.

## 🖼️ Current Output
Captura de pantalla 2024-12-19 a la(s) 9 38 20 p  m

Captura de pantalla 2024-12-19 a la(s) 9 38 55 p  m

## 🧪 Testing
I tried with different sizes, in different background positions. 

## 💬 Comments
Let me know if I can help with something else.

### CODE REVIEW

**Comment by MPSxDev** [comment]
image
can you add some margin to this component please

File: src/components/sections/HowItWorks.tsx
Comment (MPSxDev) [inline src/components/sections/HowItWorks.tsx:1]: image
can you add some margin to this component please

**Review by MPSxDev — CHANGES_REQUESTED** [review]

File: src/components/sections/HowItWorks.tsx
Comment (kimcascante) [reply src/components/sections/HowItWorks.tsx:1]: @MPSxDev done :) ! let me know

**Review by MPSxDev — APPROVED** [review]
Excellent, thanks @kimcascante

**Pull request merged**